include/pros/imu.h

```diff
@@ -28,6 +28,8 @@ namespace pros {
 namespace c {
 #endif
 
+typedef struct IMU Imu;
```

Comment:
does this even compile? there are no such structs defined in the C API. I think it is sufficient to provide that alias for the C++ API only.